recipe/meta.yaml

```diff
@@ -1,46 +1,48 @@
-{% set version = "16.7.5" %}
+{% set version = "20.0.20" %}
 
 package:
   name: virtualenv
   version: {{ version }}
 
 source:
-  url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
-  sha256: b14f2548f9d9b3db3309756069d2df9cbb8eb1cbd5f968437fc4580a6dbf19b6
-  patches:
-    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
-    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
+  git_url: https://github.com/pypa/virtualenv.git
+  git_rev: {{ version }}
```

Comment:
maybe ?
```suggestion
  url: https://pypi.io/packages/source/v/virtualenv/virtualenv-{{ version }}.tar.gz
  sha256: fd0e54dec8ac96c1c7c87daba85f0a59a7c37fe38748e154306ca21c73244637
```

Reply:
That's the first thing I tried, but it tries to rebuild itself from source to install itself in a build environment (IIUC), and that did not work.

Reply:
ah, ok then
if there is no other way, lets keep like that :grimacing: